hikari/api/rest.py

```diff
@@ -2330,21 +3107,91 @@ async def fetch_ban(
         guild: snowflakes.SnowflakeishOr[guilds.PartialGuild],
         user: snowflakes.SnowflakeishOr[users.PartialUser],
     ) -> guilds.GuildMemberBan:
-        ...
+        """Fetch a users ban.
```

Comment:
`"user's ban"` and might be worth saying something along the line of "Fetch the state of a user's ban." as "Fetch a user's ban" is a bit ambiguous.

Reply:
Bans dont have states, they are just entries. Maybe I will change it to `Fetch a ban` or something along the lines

Reply:
I mean an entry is just a state but we use the word "ban" as both a noun and a verb which might get confusing (think of "ban_user" as opposed to "create_ban" in some places but in other places fetch_ban)